lib/rules/jsx-child-element-spacing.js

```diff
@@ -39,13 +39,14 @@ const INLINE_ELEMENTS = new Set([
 
 module.exports = {
   meta: {
+    type: 'layout',
     docs: {
       description: 'Ensures inline tags are not rendered without spaces between them',
       category: 'Stylistic Issues',
       recommended: false,
       url: docsUrl('jsx-child-element-spacing')
     },
-    fixable: null,
+    // fixable: null,
```

Comment:
why is this commented out?

Reply:
The string "fixable: null" does not make sense. But maybe someone needs it, so I did not delete it.

Reply:
```suggestion
```